keyboards/4by3/info.json

```
@@ -0,0 +1,35 @@
+{
+    "keyboard_name": "4by3",
+    "url": "https://github.com/eliassjogreen/4by3",
+    "maintainer": "eliassjogreen",
+    "width": 4,
+    "height": 3,
+    "layouts": {
+        "LAYOUT_horizontal": {
+            "key_count": 12,
+            "layout": [
+                { "x": 0, "y": 0 }, { "x": 1, "y": 0 }, { "x": 2, "y": 0 }, { "x": 3, "y": 0 },
+                { "x": 0, "y": 1 }, { "x": 1, "y": 1 }, { "x": 2, "y": 1 }, { "x": 3, "y": 1 },
+                { "x": 0, "y": 2 }, { "x": 1, "y": 2 }, { "x": 2, "y": 2 }, { "x": 3, "y": 2 }
+            ]
+        },
+        "LAYOUT_vertical_right": {
+            "key_count": 12,
```

You can actually set the `width` and `height` properties per layout:

```suggestion
            "key_count": 12,
            "width": 3,
            "height": 4,
```

Reference: [info.json Format - QMK Firmware Documentation](https://docs.qmk.fm/#/reference_info_json)